texar/torch/modules/classifiers/bert_classifier.py

```diff
@@ -43,16 +43,11 @@ class BERTClassifier(ClassifierBase, PretrainedBERTMixin):
     Arguments are the same as in
     :class:`~texar.torch.modules.BERTEncoder`.
 
-    Please refer to :class:`~texar.torch.modules.pretrained.PretrainedBERTMixin`
-    for the detailed information of the supported models.
-
     Args:
         pretrained_model_name (optional): a str with the name
-            of a pre-trained model to load selected in the list of:
-            `bert-base-uncased`, `bert-large-uncased`, `bert-base-cased`,
-            `bert-large-cased`, `bert-base-multilingual-uncased`,
-            `bert-base-multilingual-cased`, `bert-base-chinese`,
-            `roberta-base-cased`, `roberta-large-cased`.
+            of a pre-trained model. Please refer to
+            :class:`~texar.torch.modules.pretrained.PretrainedBERTMixin` for
+            the detailed information of the supported models.
             If `None`, will use the model name in :attr:`hparams`.
```

Comment:
polished a bit as below:

a `str`, the name of pre-trained model (e.g., "bert-base-uncased"). Please refer to :class:`~texar.torch.modules.pretrained.PretrainedBERTMixin` for all supported models (including the standard BERT models and variants like RoBERTa).
If `None`, the model name in :attr:`hparams` is used.